### PR TITLE
login with DNS

### PR DESCRIPTION
**Pull request by ErinBecker**
Change login instructions to use DNS instead of IP address, to fit in with existing workshop administration workflow. I have tested this and confirmed that it works using the EC2 security group provided here: https://datacarpentry.org/genomics-workshop/AMI-setup/index.html.

### CODE REVIEW

File: _episodes_rmd/01-introduction.Rmd [hunk 1]
```diff
@@ -101,12 +101,12 @@ standalone software.
 
 ## Log on to RStudio Server
 
-Open a web browser and enter the IP address of your instance 
+Open a web browser and enter the DNS address you used to log in at the terminal
```
Comment (naupaka) [inline _episodes_rmd/01-introduction.Rmd]: I'd just say URL here. It's true that it's a DNS hostname, but I don't think that is important for our purposes and might just add unnecessary cognitive load?

Comment (naupaka) [reply _episodes_rmd/01-introduction.Rmd]: Seeing as most folks are familiar with URL but maybe not with DNS...

Comment (ErinBecker) [reply _episodes_rmd/01-introduction.Rmd]: Great, thanks! I'll make this change and then merge.

**Comment by naupaka** [comment]
👍

**Review by naupaka — APPROVED** [review]

**Pull request merged**